File: fix_surveys.py
import os

# Use the actual path
path = os.path.join('c:', os.sep, '\uc11c\uc6184 3.0', 'src', 'app', 'surveys', 'page.tsx')
print('Reading:', path)

with open(path, 'r', encoding='utf-8') as f:
    lines = f.readlines()

print('Total lines:', len(lines))

# Remove lines 1254 to 1511 (0-indexed), keep everything else
kept = lines[:1254] + lines[1511:]

print('Lines after removal:', len(kept))

with open(path, 'w', encoding='utf-8', newline='') as f:
    f.writelines(kept)

print('Done!')

File: fix_surveys.js
const fs = require('fs');

const filePath = 'c:\\서울3.0\\src\\app\\surveys\\page.tsx';
console.log('Reading:', filePath);

const content = fs.readFileSync(filePath, 'utf8');
const lines = content.split('\n');
console.log('Total lines:', lines.length);

// Remove lines at index 1254 through 1510 (0-indexed)
const kept = [...lines.slice(0, 1254), ...lines.slice(1511)];
console.log('Lines after:', kept.length);

fs.writeFileSync(filePath, kept.join('\n'), 'utf8');
console.log('Done!');
